docs/section4/4-1.md

```diff
@@ -0,0 +1,38 @@
+# Airside System – Rule 4-1
+
+**Rule ID:** 4-1
+ 
+**Rule Description:** Temperature Control Setpoints shall be the same for proposed design and baseline building design.
+**Rule Assertion:** P-RMR = B-RMR                                           
+**Appendix G Section:** Table G3.1 Section 4 proposed and baseline columns including exception #1 and #2. 
+**Appendix G Section Reference:** None
+**Data Lookup:** None 
+**Evaluation Context:** Evaluate each Zone  
+
+**Applicability Checks:** None
+ 
+**Function Call:**
+
+1. match_data_element()
+
+**Manual Check:** 
+
+1. If outcome is UNDETERMINED then conduct a manual check to see if Table G3.1 #4 baseline column exception #s 1 and/or 2 are applicable. If not, then check fails.
+
+**Rule Logic:**
+- For each zone in the B_RMR: `for zone_b in B_RMR..Zone:`
+    - Schedule mismatch boolean variable: `schedule_mismatch = FALSE`
+    - Get baseline thermostat_cooling_setpoint_schedule: `thermostat_cooling_stpt_sch_b = zone_b.thermostat_cooling_setpoint_schedule`
+    - Get analogous proposed thermostat_cooling_setpoint_schedule: `thermostat_cooling_stpt_sch_p = match_data_element(P_RMR,Zone,zone_b.id.id).thermostat_cooling_setpoint_schedule`
+    - Loop through each hour of the schedules and compare across the B_RMR and P_RMR, : `for x in len(thermostat_cooling_stpt_sch_b.hourly_values):`
+        - Check if schedules match each hour of the year, if there is a mismatch then set the schedule_mismatch boolean variable equal to TRUE: `if thermostat_cooling_stpt_sch_b.hourly_values(x) != thermostat_cooling_stpt_sch_p.hourly_values(x):schedule_mismatch = TRUE`
+    - After looping through the cooling thermostat schedule if schedule_mismatch = FALSE then loop through the heating thermostat schedule: `if schedule_mismatch == FALSE:`
```

Comment:
@claperle we have a schedule compare function that @xingcx wrote that could be used for this schedule compare. Would this be useful https://github.com/pnnl/ruleset-checking-tool/pull/156

Reply:
@supriyagoel I don't have strong feelings one way or the other whether we should use this function. It provides a more detailed output than just whether there is a match, as it was originally developed to help verify correlation between the baseline and proposed lighting schedules to verify whether lighting controls are modeled correctly.